lib/cdo/test_run_utils.rb

```diff
@@ -26,10 +26,110 @@ def self.run_blockly_core_tests
     end
   end
 
-  def self.run_dashboard_tests(parallel: false)
+  # Setup method to prepare test databases for parallel dashboard tests,
+  # which can be called from the QA pass (the test machine) or CI (Drone).
```

Comment:
```suggestion
  # which can be called from the QA pass (the Test environment) or CI (Drone).
```

Suggesting we use a capitalized proper noun for the Test environment.

Reply:
I've renamed this to "the chef-managed test system", for better consistency with https://github.com/code-dot-org/code-dot-org/blob/9bbfef09e67d636e56ab0fe9b7354b683023692d/lib/cdo.rb#L319-L323

I didn't see a lot of existing references to `Test environment` in our codebase, and some of the ones I did see looked ambiguous as to whether the capitalization was intentional due to being at the start of a sentence.